models/models.js

```diff
@@ -43,3 +43,20 @@ exports.fetchArticleComments = (article_id) => {
     return rows;
   });
 };
+
+exports.insertComment = (article_id, username, body) => {
+  return db
+    .query(
+      "INSERT INTO comments (article_id, author, body) VALUES($1, $2, $3) RETURNING *",
+      [article_id, username, body]
+    )
+    .then(({ rows }) => {
+      // if (rows.length === 0) {
+      //   return Promise.reject({
+      //     status: 404,
+      //     message: "article does not exist",
+      //   });
+      // }
+      return rows[0];
```

Comment:
Nice use of $ syntax to avoid SQL injection 💉
Don't forget to remove comments from your code, keep it looking nice and tidy 🧹